applications/graph/node2vec/model/random_projection.py

```diff
@@ -10,46 +10,46 @@ def random_projection(indices, num_projections, projection_dim):
     # Note: proj_indices(i) = index*projection_dim + i
     proj_indices = lbann.WeightedSum(
         indices,
-        scaling_factors=utils.str_list(projection_dim),
+        scaling_factors=projection_dim,
     )
     iota = lbann.WeightsLayer(
-        dims=utils.str_list(projection_dim),
+        dims=.projection_dim,
```

Comment:
@benson31 Is this period supposed to be at the start of the list?

Reply:
Oh, probably not.

Reply:
Fixed.